descartes_core/include/descartes_core/cart_trajectory_pt.h

```diff
@@ -160,22 +160,22 @@ struct TolerancedFrame: public Frame
   {
     Eigen::Vector3d t = a.translation();
     Eigen::Matrix3d m = a.rotation();
-    Eigen::Vector3d rpy = m.eulerAngles(2,1,0);
+    Eigen::Vector3d rxyz = m.eulerAngles(0,1,2);
```

Comment:
The axis of rotation corresponding to the roll pitch yaw denominations was not clearly defined in most of the literature that I found on this.  In order to avoid further confusing myself I renamed the variable to something indicative to the axis of rotations and the order in which they are applied.  In this case, the combined rotation 'rxyz' is the product of Rx * Ry * Rz in that specific order.